adr/README.md

```diff
@@ -0,0 +1,20 @@
+# Architecture Decision Records Folder
+This folder contains EdgeX Foundry architecture decision records (ADR).  At the root of the folder are decisions that are relevant to multiple parts of the project (aka � *cross cutting concerns*).  Sub folders contain decisions relevant to the specific area of the project and essentially set up along working group lines (security, core, application, etc.).
+## Naming and Formatting
+ADR documents are requested to follow [RFC]( https://tools.ietf.org/html/rfc2026) (request for comments) naming standard.  Specifically, authors should name their documents with a sequentially increasing integer (or serial number).  
```

Comment:
Can we drop the RFC part. I looked at that link and it doesn't provide info that was useful for naming.
Can we just say name file with leading sequentially increasing integer. i.e. `0- design XYZ`

Reply:
will make the change